simpeg/optimization.py

```diff
@@ -390,9 +393,12 @@ def evalFunction(x, return_g=False, return_H=False):
             p = self.scaleSearchDirection(self.searchDirection)
             xt, passLS = self.modifySearchDirection(p)
             if not passLS:
-                xt, caught = self.modifySearchDirectionBreak(p)
-                if not caught:
-                    return self.xc
+                if self.aggressive_stepping is False:
```

Comment:
```suggestion
                if not self.aggressive_stepping:
```

Reply:
Or `strict_decrease`? (Current behavior would is `True`

Reply:
or maybe `require_decrease` (with the default being `True`). That might be a touch more intuitive than `strict`?

Reply:
`require_decrease` sounds the better option to me.